src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -252,10 +252,13 @@ public Response importDataset( String jsonBody, @PathParam("identifier") String
             if ( version.getVersionState() == null ) {
                 version.setVersionState(DatasetVersion.VersionState.DRAFT);
             }
-            
-            if ( nonEmpty(pidParam) ) {
+
+            if (nonEmpty(pidParam)) {
+                if (GlobalId.verifyImportCharacters(pidParam)) {
+                    return badRequest("PID parameter contains characters not allowed. They must match this regex on import: " + GlobalId.ALLOWED_CHARACTERS);
```

Comment:
May be good to say in this "Not allowed by the Dataverse application" for clarity, as they are allowed by the provider.

Reply:
Other than that, this looks like an informative error message :)